### PR TITLE
Add Integration Test for Zouwu AutoTS

### PR DESCRIPTION
**Pull request by PeterQiu0516**
Fix issue #3343.

Add Integration Test for Zouwu AutoTS.

### CODE REVIEW

File: pyzoo/zoo/examples/run-example-test-ray-horovod-pytorch.sh [hunk 1]
```diff
@@ -31,5 +31,15 @@ now=$(date "+%s")
 time2=$((now-start))
 echo "zouwu tcmf example tests finished"
 
+echo "Start zouwu autots tests"
```
Comment (shane-huang) [inline pyzoo/zoo/examples/run-example-test-ray-horovod-pytorch.sh:34]: move this test to https://github.com/intel-analytics/analytics-zoo/blob/master/pyzoo/zoo/examples/run-example-test-ray-horovod-pytorch-integration.sh

File: pyzoo/zoo/examples/run-example-test-ray-horovod-pytorch-integration.sh [hunk 2]
```diff
@@ -21,8 +21,24 @@ fi
 now=$(date "+%s")
 time1=$((now-start))
 
+echo "#2 Start zouwu autots tests"
+start=$(date "+%s")
+# run example
```
[the remark this answers is not in the document]
Comment (yushan111) [reply pyzoo/zoo/examples/run-example-test-ray-horovod-pytorch-integration.sh:26]: You can also use $ANALYTICS_ZOO_HOME/dist/bin/data/NAB/nyc_taxi/get_nyc_taxi.sh to download the dataset

File: pyzoo/zoo/zouwu/examples/quickstart/zouwu_autots_nyc_taxi.py [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    init_orca_context(cluster_mode=args.cluster_mode,
+                      cores=args.cores,
+                      num_nodes=num_nodes,
+                      memory=args.memory,
+                      init_ray_on_spark=True
+                      )
+
+
+    # recommended to set it to True when running Analytics Zoo in Jupyter notebook 
+    OrcaContext.log_output = True # (this will display terminal's stdout and stderr in the Jupyter notebook).
```
Comment (yushan111) [inline pyzoo/zoo/zouwu/examples/quickstart/zouwu_autots_nyc_taxi.py:62]: We may not need that in python script

**Review by yushan111 — COMMENTED** [review]
Besides, you can add the auto test in "pyzoo/zoo/examples/run-example-tests-yarn-integration.sh" instead of "run-example-test-ray-horovod-pytorch-integration.sh" since we may not need horovod to run the example.

You can test with http://10.239.47.210:18888/view/ZOO-PR/job/ZOO-PR-Python-integration-test/

File: pyzoo/zoo/examples/run-example-test-ray-horovod-pytorch.sh (hunk 1 above)
Comment (yushan111) [reply pyzoo/zoo/examples/run-example-test-ray-horovod-pytorch.sh]: Sorry I missed that. I think https://github.com/intel-analytics/analytics-zoo/blob/master/pyzoo/zoo/examples/run-example-tests-yarn-integration.sh is fine. And the test environment already contains pytorch.

File: pyzoo/zoo/examples/run-example-test-ray-horovod-pytorch-integration.sh [hunk 4]
```diff
@@ -21,8 +21,24 @@ fi
...
+python ${ANALYTICS_ZOO_ROOT}/pyzoo/zoo/zouwu/examples/quickstart/zouwu_autots_nyc_taxi.py
+exit_status=$?
+if [ $exit_status -ne 0 ];
+then
+    clear_up
+    echo "ray horovod pytorch failed"
```
[the remark this answers is not in the document]
Comment (PeterQiu0516) [reply pyzoo/zoo/examples/run-example-test-ray-horovod-pytorch-integration.sh:32]: Fixed.

File: pyzoo/zoo/examples/run-example-test-ray-horovod-pytorch-integration.sh [hunk 5]
```diff
@@ -21,8 +21,24 @@ fi
 now=$(date "+%s")
 time1=$((now-start))
 
+echo "#2 Start zouwu autots tests"
+start=$(date "+%s")
+# run example
+python ${ANALYTICS_ZOO_ROOT}/pyzoo/zoo/zouwu/examples/quickstart/zouwu_autots_nyc_taxi.py
```
[the remark this answers is not in the document]
Comment (PeterQiu0516) [reply pyzoo/zoo/examples/run-example-test-ray-horovod-pytorch-integration.sh:27]: Fixed.

File: pyzoo/zoo/examples/run-example-test-ray-horovod-pytorch.sh (hunk 1 above)
Comment (PeterQiu0516) [reply pyzoo/zoo/examples/run-example-test-ray-horovod-pytorch.sh]: Fixed.

File: pyzoo/zoo/examples/run-example-test-ray-horovod-pytorch-integration.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (PeterQiu0516) [reply pyzoo/zoo/examples/run-example-test-ray-horovod-pytorch-integration.sh:26]: Fixed.

File: pyzoo/zoo/zouwu/examples/quickstart/zouwu_autots_nyc_taxi.py (hunk 3 above)
Comment (PeterQiu0516) [reply pyzoo/zoo/zouwu/examples/quickstart/zouwu_autots_nyc_taxi.py]: Fixed.

File: pyzoo/zoo/zouwu/examples/quickstart/zouwu_autots_nyc_taxi.py [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+from zoo.orca import OrcaContext
+from zoo.zouwu.autots.forecast import AutoTSTrainer
+from zoo.zouwu.config.recipe import LSTMGridRandomRecipe
+from zoo.zouwu.autots.forecast import TSPipeline
+import os
+import argparse
+import tempfile
+
+
+os.environ["JAVA_HOME"] = "/usr/lib/jvm/java-8-openjdk-amd64"
```
[the remark this answers is not in the document]
Comment (PeterQiu0516) [reply pyzoo/zoo/zouwu/examples/quickstart/zouwu_autots_nyc_taxi.py:28]: Fixed.

**Comment by PeterQiu0516** [comment]
Jenkins passed: http://10.239.47.210:18888/job/ZOO-PR-Validation/5446/

**Comment by PeterQiu0516** [comment]
Integration test passed: http://10.239.47.210:18888/view/ZOO-PR/job/ZOO-PR-Python-integration-test-horovod/10/

**Pull request merged**